### ISSUE TITLE
Option to turn timer on/off

### ISSUE THREAD

**Issue by GoogleCodeExporter** [thaindq/opensudoku-android#5]
```
Option to turn timer on/off.  When off, display app name.
```

Original issue reported on code.google.com by `calam...@gmail.com` on 5 May 2009 at 3:05

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 6 May 2009 at 11:53

**Comment by GoogleCodeExporter** [comment]
```
Added option to hide time during gameplay (however gameplay time is still 
tracked).
It will be released in 0.6.1.
```

Original comment by `romario...@gmail.com` on 7 May 2009 at 6:28
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 1 Oct 2009 at 9:03
- Changed state: **Verified**
